test/e2e/predictor/test_huggingface.py

```diff
@@ -339,3 +339,57 @@ async def test_huggingface_v2_text_embedding(rest_v2_client):
     assert res.outputs[0].data == huggingface_text_embedding_expected_output
 
     kserve_client.delete(service_name, KSERVE_TEST_NAMESPACE)
+
+@pytest.mark.llm
+@pytest.mark.asyncio(scope="session")
+async def test_huggingface_v2_sequence_classification_with_probabilities(rest_v2_client):
+    service_name = "hf-bert-sequence-v2"
+    protocol_version = "v2"
+    predictor = V1beta1PredictorSpec(
+        min_replicas=1,
+        model=V1beta1ModelSpec(
+            model_format=V1beta1ModelFormat(
+                name="huggingface",
+            ),
+            protocol_version=protocol_version,
+            args=[
+                "--model_id",
+                "textattack/bert-base-uncased-yelp-polarity",
+                "--model_revision",
+                "a4d0a85ea6c1d5bb944dcc12ea5c918863e469a4",
+                "--tokenizer_revision",
+                "a4d0a85ea6c1d5bb944dcc12ea5c918863e469a4",
+                "--backend",
+                "huggingface",
+                "--return_probabilities",
+            ],
+            resources=V1ResourceRequirements(
+                requests={"cpu": "1", "memory": "2Gi"},
+                limits={"cpu": "1", "memory": "4Gi"},
+            ),
+        ),
+    )
+
+    isvc = V1beta1InferenceService(
+        api_version=constants.KSERVE_V1BETA1,
+        kind=constants.KSERVE_KIND,
+        metadata=client.V1ObjectMeta(
+            name=service_name, namespace=KSERVE_TEST_NAMESPACE
+        ),
+        spec=V1beta1InferenceServiceSpec(predictor=predictor),
+    )
+
+    kserve_client = KServeClient(
+        config_file=os.environ.get("KUBECONFIG", "~/.kube/config")
+    )
+    kserve_client.create(isvc)
+    kserve_client.wait_isvc_ready(service_name, namespace=KSERVE_TEST_NAMESPACE)
+
+    res = await predict_isvc(
+        rest_v2_client,
+        service_name,
+        "./data/bert_sequence_classification_v2.json",
+    )
+    assert res.outputs[0].data == ['{0: -2.152204, 1: 2.5094059}']
```

Comment:
You have to limit the precision like it is done here https://github.com/kserve/kserve/blob/master/python/huggingfaceserver/tests/test_output.py